Makefile

```diff
@@ -19,12 +19,16 @@ generate:
 
 .PHONY: test
 test: verify
-	go test ./...
+	go test ./cmd/... ./pkg/...
 
 .PHONY: release-local
 release-local:
 	MANIFESTS=$(shell mktemp -d) hack/release-local.sh
 
+.PHONY: test-e2e
+test-e2e:
+	hack/test-e2e.sh
```

Comment:
How about instead:

1. Use build tags for the e2e tests, e.g. 
```go
// +build e2e

package e2e
```

2. For unit tests, run `go test -v ./...` as usual (this will exclude `e2e` tagged tests)
3. Remove the `test-e2e.sh` script which isn't really pulling its weight, and replace it with this in the target:

```make
KUBERNETES_CONFIG="$KUBECONFIG" go test -v -tags e2e ./...
```

Reply:
Done.  Thanks!